src/components/Balance.tsx

```diff
@@ -114,6 +119,22 @@ export const Balance = ({
           callback: onSend,
         };
         break;
+      case AssetActionId.SEP6_DEPOSIT:
+        props = {
+          ...defaultProps,
+          title: `SEP-6 deposit ${balance.assetCode} (with Trusted Asset)`,
+          description: `Start SEP-6 deposit of trusted asset ${balance.assetCode}?`,
+          callback: () => handleSep6Deposit(balance),
+        };
+        break;
+      case AssetActionId.SEP6_WITHDRAW:
+        props = {
+          ...defaultProps,
+          title: `SEP-6 withdrawal ${balance.assetCode}`,
+          description: `Start SEP-6 withdrawal of ${balance.assetCode}?`,
+          callback: () => handleSep24Withdraw(balance),
+        };
+        break;
```

Comment:
this PR covers SEP-6 deposit of a trusted asset. A follow up PR will cover untrusted assets using claimable balances